src/libasr/ASR.asdl

```diff
@@ -164,6 +168,7 @@ abi                   -- External     ABI
     | GFortranModule  --   Yes        GFortran
     | BindC           --   Yes        C
     | BindPython      --   Yes        Python
+    | BindJS          --   Yes        Javascript
```

Comment:
I added a new decorator for calls to Javascript functions. We previously used the (BindC + implementation=interface) for recognizing the functions to be imported. But it seems some functions from the numpy intrinsic module use these exact parameters (BindC + interface) and thus we are unable to distinguish between these numpy intrinsic functions and the functions that call into Javascript.

Reply:
I think that's fine. The same should be done in LFortran.